code/game/machinery/requests_console.dm

```diff
@@ -189,7 +196,24 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 			pass = 1
 		if(pass)
 			screen = RCS_SENTPASS
-			message_log += "<B>Message sent to [recipient]</B><BR>[message]"
+			if(recipient in list("Atmospherics","Mechanic","Engineering","Chief Engineer's Desk"))
+				radiochannel = "Engineering"
+			if(recipient in list("Warden","Security","Brig Medbay","Head of Security's Desk"))
+				radiochannel = "Security"
+			if(recipient in list("Bar","Chapel","Kitchen","Hydroponics","Janitorial"))
+				radiochannel = "Service"
+			if(recipient in list("Virology","Chief Medical Officer's Desk","Medbay",))
+				radiochannel = "Medical"
+			if(recipient in list("Blueshield","NT Representative","Head of Personnel's Desk","Captain's Desk","Bridge"))

```

Comment:
Would it not be better to just generate and define these lists up front, rather than having the game temporarily create and then chuck these lists each time? Will this not have a performance impact?